src/config/tracker.js

```diff
@@ -11,6 +11,14 @@ const trackerRaduisMapping = {
   9: [5, 5, 5, 5, 5, 7, 7, 7, 7, 7, 9, 12.5, 14, 15, 15, 15, 15],
 };
 
+const getTypeIndex = (type, types) => {
```

Comment:
AFAIK, no need to provide `types` as a param, just move `export const types = ...` above this function.